spec/ParseLiveQuery.spec.js

```
@@ -1308,4 +1308,128 @@ describe('ParseLiveQuery', function () {
+        classNames: ['TestObject'],
+      },
+      startLiveQueryServer: true,
+      verbose: false,
+      silent: true,
+    });
+    const object = new TestObject();
+    object.set('foo', 'bar');
+    await object.save();
+
+    Parse.Cloud.beforeLiveQueryEvent('TestObject', async req => {
+      expect(req.object.get('foo')).toBe('baz');
```

ESLint

[error] 'expect' is not defined.

(no-undef)